migration-from-froala.md

```diff
@@ -0,0 +1,485 @@
+---
+layout: default
+title: Migrating from Froala to TinyMCE
+title_nav: Migrating from Froala
+description: Upgrading your rich text editor from Froala Editor v3 to TinyMCE 5.
+keywords: migration considerations premigration pre-migration froala
+---
+
+This page covers the basic steps for migrating from Froala Editor 3 to TinyMCE 5.
+Procedures and links to our documentation have been included to assist with your migration to TinyMCE.
+
+**Covered in this section:**
```

Comment:
These could all be just anchor based links, instead absolute links. eg `[Migrating a Basic Froala Configuration to TinyMCE](#migratingabasicfroalaconfigurationtotinymce)`